ios/RNZendeskChat.h

```diff
@@ -3,5 +3,11 @@
 #import <React/RCTBridgeModule.h>
 
 @interface RNZendeskChat : NSObject<RCTBridgeModule>
+@end
+
+
+@interface NavigationControllerWithCompletion : UINavigationController
+@property (nonatomic, copy) RCTResponseSenderBlock completion;
```

Comment:
```suggestion
@property (nonatomic, copy, nullable) RCTResponseSenderBlock completion;
```
Just because of the check you did before calling it, but also because I'll set it `nil` after its invocation because of [this](https://reactnative.dev/docs/native-modules-ios#callbacks):
> A native module is supposed to invoke its callback only once. It can, however, store the callback and invoke it later. This pattern is often used to wrap iOS APIs that require delegates— see [RCTAlertManager](https://github.com/facebook/react-native/blob/main/packages/react-native/React/CoreModules/RCTAlertManager.mm) for an example. If the callback is never invoked, some memory is leaked.

What do you think?